### ISSUE TITLE
Value from bind:value in <Input type="number"/"range"> is "string" and not "number"

### ISSUE THREAD

**Issue by marzeq** [bestguy/sveltestrap#328]
In this example code using vanilla Svelte and TypeScript:
```svelte
<script lang="ts">
    import { Input } from "sveltestrap"
    let someVariable: number
</script>

<Input type="number" bind:value={someVariable} /> <!-- Type 'number' is not assignable to type 'string'. -->
```

the following error gets thrown by the TypeScript compiler: "Type 'number' is not assignable to type 'string'.", whereas if I would use the vanilla \<input\> element everything would be fine and the compiler would understand that the value is actually a number.

Environment information:
Sveltestrap 5.4.0
TypeScript 4.3.5
Svelte 3.38.3
Linux (pop os) (kernel 5.11.0-7620)

**Comment by bestguy** [comment]
Thanks, I'm assuming the d.ts here is wrong: https://github.com/bestguy/sveltestrap/blob/master/src/Input.d.ts

Though also assuming string | number is not sufficient.  Open to suggestions and will take a look this week

**Comment by marzeq** [comment]
Thanks!

**Comment by bestguy** [comment]
Hi @marzeq ,

This should be corrected by #329 , and I've published a prerelease version `sveltestrap@5.4.1-0` if you could try and confirm it corrects your issue.  It looks okay to me locally.

**Comment by bestguy** [comment]
Released in [sveltestrap@5.5.0](https://github.com/bestguy/sveltestrap/releases/tag/v5.5.0)
